### PR TITLE
Updating item interaction keys

### PR DESCRIPTION
**Pull request by SaphiraSkies**
These need to match our glossary format. Also removed item template since it's no longer needed.

### CODE REVIEW

File: Items/ladder.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "description": "It's a small aluminum ladder. Useful for reaching high places.",
     "secondaryDescription": "You drop the ladder underneath the attic hatch. You open the hatch and climb up.",
     "itemInteractions": {
-        "Ladder": "None",
+        "ladder": "None",
         "Flashlight": "None",
```
Comment (CharlesBeise) [inline Items/ladder.json:7]: The rest of ladder.json needs to be updated.

**Review by CharlesBeise — CHANGES_REQUESTED** [review]
Just need to update ladder.json, otherwise looks good!

File: Items/ladder.json (hunk 1 above)
Comment (SaphiraSkies) [reply Items/ladder.json]: Whoops, got distracted lol. Done.

**Review by CharlesBeise — APPROVED** [review]
Looks good! Thanks for fixing this! (And by "this" I mean all of the item files)

**Pull request merged**